app/src/main/java/org/mozilla/rocket/content/travel/ui/TravelCityViewModel.kt

```diff
@@ -227,5 +228,6 @@ class TravelCityViewModel(
 
     companion object {
         private const val LOAD_MORE_HOTELS_THRESHOLD = 15
+        private const val VIDEO_QUERY_PARAM = "%s+%s"
```

Comment:
Suggest to call it `VIDEO_QUERY_PATTERN` or `VIDEO_QUERY_PARAM_PATTERN` to be more descriptive.